apps/journeys-admin/src/components/SingleJourney/CardOverview/CardOverview.tsx

```diff
@@ -1,25 +1,91 @@
 import { ReactElement } from 'react'
-import { Button, Box, Typography } from '@mui/material'
+import { Typography, Fab, Card, Box } from '@mui/material'
 import EditIcon from '@mui/icons-material/Edit'
+import AddToPhotosIcon from '@mui/icons-material/AddToPhotos'
+
+import { BlockFields_StepBlock as StepBlock } from '../../../../__generated__/BlockFields'
+import { CardPreview } from '../../CardPreview'
+import { TreeBlock } from '@core/journeys/ui'
+import { useBreakpoints } from '@core/shared/ui'
 
 export interface CardOverviewProps {
   slug: string
+  blocks: Array<TreeBlock<StepBlock>>
 }
 
-const CardOverview = ({ slug }: CardOverviewProps): ReactElement => {
+const CardOverview = ({ slug, blocks }: CardOverviewProps): ReactElement => {
+  const breakpoints = useBreakpoints()
+
+  const stepBlockLength =
+    blocks?.filter((block) => block.__typename === 'StepBlock').length ?? 0
+
+  const cardNumber =
+    stepBlockLength === 1
+      ? `${stepBlockLength} card`
+      : `${stepBlockLength} cards`
+
   return (
-    <Box>
-      <Typography variant="h4">Cards</Typography>
-      {/* <StoryCards /> goes here */}
-      <Button
-        variant="contained"
-        startIcon={<EditIcon />}
-        sx={{ backgroundColor: '#C52D3A', borderRadius: '20px' }}
-        href={`/journeys/${slug}/edit`}
-      >
-        Edit
-      </Button>
-    </Box>
+    <>
+      {stepBlockLength > 0 ? (
+        <>
+          <CardPreview steps={blocks} />
+
+          <Fab
+            variant="extended"
+            size="medium"
+            sx={{
+              position: 'absolute',
+              bottom: 3,
+              right: 4
+            }}
+            color="primary"
```

Comment:
```suggestion
            color="primary"
```
Is the default color not primary?

Reply:
Nope, its like a grey without this